pkgs/leak_tracker/lib/src/leak_tracking/_leak_filter.dart

```diff
@@ -40,7 +40,7 @@ class _PhaseLeakFilter {
         return _shouldReport(
           leakType,
           record,
-          phase.ignoredLeaks.notGCed,
+          phase.ignoredLeaks.experimentalNotGCed,
```

Comment:
Is 'not GCed' experimental? Consider not including the status in the name or perhaps suffixing it instead.

Reply:
Yes, MVP does not include notGCed, while we want to pilot it with Flutter and early adopters. 
Why postfix is better? 
Prefix is consistent with how it is done in Flutter: https://github.com/flutter/flutter/blob/b45cd231739b3b141506bc218d71b78a6d4b1f24/packages/flutter_test/lib/src/widget_tester.dart#L151

Reply:
I would have chosen to suffix - the order and discovery is the same w/ or w/o the 'experimental' tag, but consistency w/ flutter makes sense.